walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -117,4 +95,66 @@
     return timeRanges;
   }
 
+  /**
+   * Returns a list containing all the TimeRanges that the given collection of attendees are available for given
+   * a collection of their events for that day.
+   */
+  private List<TimeRange> determineAvailableTime(Collection<Event> events, Collection<String> attendees, long meetingDuration) {
+    List<TimeRange> availableTime = new ArrayList<>();
+    Iterator<TimeRange> unavailableTimeIterator = determineUnavailableTime(events, attendees).iterator();
+    TimeRange currentEvent = (unavailableTimeIterator.hasNext()) ?  unavailableTimeIterator.next() : null;
+    if (currentEvent != null) {
```

Comment:
[Nit] this could actually be made a positives condition as well.  Even better you can use an early return

`if (currentEvent == null) {
  return ...
}`